ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Desktop/d3d.cpp

```diff
@@ -0,0 +1,6 @@
+#include "Graphics_Systems/OpenGL-Common/shader.h"
+#include <string>
+
+namespace enigma {
+  std::string texColorString = "TexColor = texture( en_TexSampler, v_TextureCoord.st ) * v_Color;";

```

Comment:
This actually needs to be `texture2D` for OpenGL1/GLES2, and `texture` for GL3/GLES3.
https://stackoverflow.com/a/12309212

Reply:
@RobertBColton I think I will look into this after GSoC